CHANGELOG.md

```diff
@@ -6,7 +6,60 @@ document.
 
 ## Unreleased / Beta / In Rust Nightly
 
-[83e42a23...master](https://github.com/rust-lang/rust-clippy/compare/83e42a23...master)
+[435a8ad8...master](https://github.com/rust-lang/rust-clippy/compare/435a8ad8...master)
+
+## Rust 1.71
+Current beta, released 2023-07-13
```

Comment:
Can you add one "Disclaimer" line here, that from now on the Clippy CHANGELOG will be kept shorter and only focus on the most relevant changes?

Reply:
Sure! Do we maybe want to create an issue, in case someone wants to leave feedback regarding this change? The link can be removed later, but it might be good to have something like that for one release. :)

Reply:
Could link to #10847

Reply:
I've now added this disclaimer:

> We're trying out a new shorter changelog format, that only contains significant changes. You can checkout the list of merged pull requests for a list of all changes. If you have any feedback related to the new format, please share it in [#10847](https://github.com/rust-lang/rust-clippy/issues/10847)

and also bumped the *"Current beta, released 2023-07-13"* note :)